### DISCUSSION TITLE
debain12 安装ClamAV后面板显示未启动

### DISCUSSION THREAD

**Discussion by mjsoftking** [1Panel-dev/1Panel#7610] [Q&A]
![image](https://github.com/user-attachments/assets/9a9fd52f-7662-4ec4-baa8-7931978698c5)

命令查询状态是 active (running) 的
![image](https://github.com/user-attachments/assets/18933c7e-9b7a-44cc-9814-44a3ef53d6da)
![image](https://github.com/user-attachments/assets/e7315374-78c5-41f9-9022-942dc2c5f6a8)

**Comment by ssongliu** [comment]
麻烦提供一下 systemctl is-active clamav-daemon.service 截图

**Reply by mjsoftking** [reply]
![image](https://github.com/user-attachments/assets/b4c47ffd-1c68-4359-8b7f-326518bff35e)

![image](https://github.com/user-attachments/assets/b2ab77ba-d1b9-4442-be29-d6f497849f12)

**Reply by mjsoftking** [reply]
![image](https://github.com/user-attachments/assets/0770e78d-2fc3-45d2-9d08-0b83206f4901)

**Comment by ssongliu** [comment]
是按照官方文档装的吗？
麻烦再截图看一下
`systemctl is-enabled clamd@scan.service`
`systemctl is-enabled clamav-daemon.service`

**Reply by mjsoftking** [reply]
是按照官网debain的教程安装的
![image](https://github.com/user-attachments/assets/f0f7ffc4-973b-4f6e-919f-d76ac96c969e)

**Reply by mjsoftking** [reply]
![image](https://github.com/user-attachments/assets/3d53ea8c-eede-463b-af95-7f3325bb95af)

**Comment by mjsoftking** [comment]
另外
![image](https://github.com/user-attachments/assets/9eb2c815-c381-4de0-bbf7-f1c297867de8)
跳转地址不正确

**Reply by ssongliu** [reply]
那大概率是 clamdscan 命令了， which clamdscan 和 clamdscan --version 看看

**Reply by mjsoftking** [reply]
![image](https://github.com/user-attachments/assets/d433a39f-ab66-4de4-9505-e79932376df3)

**Reply by mjsoftking** [reply]
若有需要，可以给你提供服务器及1panel的地址，随意调试看看，这个机器试用期到明天中午左右

**Reply by ssongliu** [reply]
可以，麻烦添加一下我吧
![WechatIMG375](https://github.com/user-attachments/assets/157b7c6e-706b-4e6a-afcc-2be208572599)

**Reply by mjsoftking** [reply]
添加了
